### PR TITLE
DC-2251: Add Welsh templates

### PR DESCRIPTION
**Pull request by muralishub**
- rescindedMessageAlert_cy
- verificationReminder_cy
- tax_estimate_message_alert_cy
- annual_tax_summaries_message_alert_cy
- newMessageAlert_cy

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala [hunk 1]
```diff
@@ -30,7 +32,10 @@ object SalutationHelper {
       }.mkString
     }
 
-  def salutationFrom(params: Map[String, Any]): String = {
+  def salutationFrom(params: Map[String, Any], salutationText: String = EnglishSalutation): String = {
```
Comment (lxol) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala]: It is not clear what is the purpose of the new argument `salutationText`. I didn't find any single usage.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala [hunk 2]
```diff
@@ -63,8 +68,8 @@ object SalutationHelper {
     )
 
     salutationParams match {
-      case List(Some(fullName)) => s"Dear $fullName"
-      case _ => "Dear Customer"
+      case List(Some(fullName)) => s"$EnglishSalutation $fullName"
```
Comment (lxol) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala]: There will be only English version of `def saluteFullName`. Is that by design?

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/ats/AtsTemplates.scala [hunk 3]
```diff
@@ -36,6 +37,15 @@ object AtsTemplates {
       plainTemplate = txt.annualTaxSummariesMessageAlert.f,
       htmlTemplate = html.annualTaxSummariesMessageAlert.f,
       priority = Some(MessagePriority.Background)
+    ),
+    MessageTemplate.create(
+      templateId = "annual_tax_summaries_message_alert_cy",
```
Comment (lxol) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/ats/AtsTemplates.scala]: I thought we are going to use `_cy_` suffix as we have already some templates with `_cy` suffixes.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala (hunk 1 above)
Comment (muralishub) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala]: Thank you I will change it

**Pull request merged**